### ISSUE TITLE
Usage of the "FASTQ" data analysis option for non-RML samples

### ISSUE THREAD

**Issue by Vince-janv** [Clinical-Genomics/cg#1420]
## Description

The usage of the "FASTQ" option for data analysis is used in most cases by cust001 for their RML samples. However in some cases other customers use it for samples ordered directly via the orderportal as "Type: sequencing of clinical samples".

As I've understood it these samples have a case linked to them, but It's assigned to cust000. If the samples are going through a WGS prep, and are not tumours, the case will have data analysis: mip-dna, if not it will have fastq. This is because a mip analysis is needed for a comparison with the MAF results.

I believe this to be quite confusing and needs to be simplified for clinical-genomics/project-planning#368 to work as intended. 

## Suggested solution

As with every other sample at CG each sample should be linked to at least one case associated with the same customer. This case should have data analysis: fastq. 

If a mip-analysis is needed (ie a WGS case that is not a tumour) a second case can be created associated with cust000 and used to compare with MAF. As this is an internal process, using cust000 makes sense here.

## This can be closed when

Each fastq sample should be linked to at least one case associated with the same customer and the data analysis option should be fastq.

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by patrikgrenfeldt** [comment]
@Vince-janv Could you add labels for your view on _urgency_ and _gain_?

**Comment by Vince-janv** [comment]
@patrikgrenfeldt Sure. In and of itself it's not very urgent, but it blocks another PR I'm working on. But if you agree that the suggested change is a good solution, then I can probably open a new PR and fix this myself.
